### DISCUSSION TITLE
Numexpr on dask

### DISCUSSION THREAD

**Discussion by marcelned** [dask/dask#8098] [General] [answered]
Numexpr uses multi-threading to achieve better performance.

Will this resource contention on used with dask dataframe to parallelise workflows?

**Comment by douglasdavis** [comment, marked as answer]
Since pandas will sometimes use numexpr if it's available, Dask may compete with numexpr for thread usage with some dask.dataframe workloads. The dask.dataframe documentation suggests limiting numexpr to a single thread; see e.g. [dask.dataframe.DataFrame.query](https://docs.dask.org/en/latest/generated/dask.dataframe.DataFrame.query.html)

**Reply by marcelned** [reply]
Thanks @douglasdavis . Concise and informative!
